src/System.Drawing.Common/src/System/Drawing/Bitmap.cs

```diff
@@ -20,11 +19,8 @@ namespace System.Drawing
     /**
      * Represent a bitmap image
      */
-    [
-    Serializable,
-    ComVisible(true)
-    ]
-    public sealed class Bitmap : Image
+    [ComVisible(true)]
```

Comment:
I think we've removed [ComVisible] from code elsewhere in corefx as well, though I'd need to double-check.